docs/projects/P2/1_teamprocess.md

```diff
@@ -0,0 +1,172 @@
+- **Dependencies** among tasks (if any)
+- Initial **assignments** for team members
+- **Milestones** to indicate which sprint the issue should be completed in (Sprint 1, Sprint 2, Stretch)
+- **Acceptance criteria** that describes how you know this task is done - think about how you will test this feature
+
+You should actively assign and move these issues across your Kanban board as your team makes progress. 
+
+!!! note "A Note on Grading"
+    We **will not** assess how accurately you predicted your development process, nor will we be giving points based on the complexity or quality of your changes. The focus of our evaluations will be on how you decompose the problem, how you respond to unexpected circumstances, and how you analyze and reflect on your experience later on.
+
+    We **will** check your development progress at the end of each sprint. Please be proactive in your planning to ensure that you make notable progress in both of your sprints. We highly recommend reading ahead on what our expectations are for the end of each sprint as you plan your milestone goals.
+
+Include a link to your GitHub board in your Gradescope submission. If you are unsure about your planned changes, you can ask for feedback from a member of the course staff!
+
+
+### Extra Credit (8 pts)
+Getting to know your colleagues in a friendly context can often lead to more effective collaboration; for example, healthy teams often get lunch together. To incentivize this, we will give your team extra credit for this assignment if you meet for a team bonding experience outside of a working session.
+
+You might want to eat together, go out for boba, or hold a board game session. If someone on your team is not feeling well, you may also do a virtual activity such as an online gaming session (Drawphone, Skribbl.io, etc.) or social “Zoom lunch”. 
+
+To receive extra credit, share the photo or screenshot of your team activity with your TA mentors by sending it in your team Slack channel before the deadline. We encourage you to do these types of meetings often throughout the semester!
+
+
+## Grading
+
+To receive full credit for the teamwork contract, we expect:
+
+- [ ] All sections listed above are addressed in a roughly 1-2 page PDF document submitted to Gradescope
+- [ ] Document demonstrates a clear process outline that was discussed between and agreed upon by the teammates
+- [ ] All group members’ signatures at the end of the document
+
+To receive full credit for the project planning, we expect:
+
+- [ ] A GitHub project board linked to your team repository with:
+    - [ ] A User Story column containing at least two user stories per group member that satisfy the guidelines outlined in the sections above and in lecture
+    - [ ] A Backlog column containing a series of GitHub issues describing the feature(s) that the team will tackle. Each issue makes use of GitHub features to denote all of the required information listed above
+
+## Resources & Documentation
```

Comment:
Do we plan on adding anything here? can delete otherwise

Reply:
Added to 2B/C, removed from 2A